Moblin/View/Stream/StreamOverlayView.swift

```diff
@@ -62,9 +64,9 @@ struct StreamOverlayView: View {
                 }
                 .opacity(model.showChatMessages ? 1 : 0)
                 if !model.showChatMessages {
-                    ChatWarning(message: String(localized: "Chat is hidden"))
+                    ChatWarning(message: String(localized: "Chat is hidden"), icon: "exclamationmark.triangle.fill", iconColor: .yellow)
                 } else if model.chatPaused {
-                    ChatWarning(message: String(localized: "Chat is paused"))
+                    ChatWarning(message: String(format: String(localized: "%d new Messages"), model.pausedChatPosts.count), icon: "pause", iconColor: .white)
```

Comment:
I think `"{model.pausedChatMessages} new messages"` is cleaner.

Reply:
Didn't know that syntax. I'm total beginner when it come to swift. Thanks